lib/soapy_cake/response.rb

```diff
@@ -22,6 +22,14 @@ def to_enum
       end
     end
 
+    def to_xml
```

Comment:
I'd like to prevent parsing the whole XML as well. My initial attempt was to just check the string for ``<row_count>0</row_count>``, but this also won't work since `row_count` is always the grand total, not the total of this request... So I'm open for ideas here to improve.